### PR TITLE
\hypertarget{JUMP}{JUMP}, \hypertarget{JUMPI}{JUMPI} and \label{eq:mu_pc}

### PR DESCRIPTION
**Pull request by jamesray1**

### CODE REVIEW

File: Paper.tex [hunk 1]
```diff
@@ -1020,7 +1020,7 @@ \subsection{The Execution Cycle}
 
 The gas is reduced by the instruction's gas cost and for most instructions, the program counter increments on each cycle, for the three exceptions, we assume a function $J$, subscripted by one of two instructions, which evaluates to the according value:
 \begin{eqnarray}
-\quad \boldsymbol{\mu}'_{g} & \equiv & \boldsymbol{\mu}_{g} - C(\boldsymbol{\sigma}, \boldsymbol{\mu}, I) \\
+\quad \boldsymbol{\mu}'_{g} & \equiv & \boldsymbol{\mu}_{g} - C(\boldsymbol{\sigma}, \boldsymbol{\mu}, I) \hypertarget{JUMP}{JUMP} \hypertarget{JUMPI}{JUMPI} \label{eq:mu_pc}\\
```
Comment (pirapira) [inline Paper.tex]: I'd expect `JUMP` and `JUMPI` jumps to those in the instruction table.

File: Paper.tex [hunk 2]
```diff
@@ -951,7 +951,8 @@ \subsubsection{Exceptional Halting}
 \boldsymbol{\mu}_g < C(\boldsymbol{\sigma}, \boldsymbol{\mu}, I) \quad \vee \\
 \mathbf{\delta}_w = \varnothing \quad \vee \\
 \lVert\boldsymbol{\mu}_\mathbf{s}\rVert < \mathbf{\delta}_w \quad \vee \\
-( w \in \{ \text{\small JUMP}, \text{\small JUMPI} \} \quad \wedge \\ \quad \boldsymbol{\mu}_\mathbf{s}[0] \notin D(I_\mathbf{b}) ) \quad \vee \\
+( w \in \{ \text{\small JUMP}, \text{\small
```
Comment (pirapira) [inline Paper.tex]: Why did `JUMPI` disappear?

Comment (jamesray1) [reply Paper.tex]: Must be an oversight.

File: Paper.tex [hunk 3]
```diff
@@ -2082,11 +2082,11 @@ \subsection{Instruction Set}
 0 & \text{otherwise}
 \end{cases}$ \\
 \midrule
-0x56 & {\small JUMP} & 1 & 0 & Alter the program counter. \\
+\linkdest{JUMP}{JUMP}0x56 & {\small JUMP} & 1 & 0 & Alter the program counter. \\
```
Comment (pirapira) [inline Paper.tex]: The second argument should be empty.

File: Paper.tex [hunk 4]
```diff
@@ -2082,11 +2082,11 @@ \subsection{Instruction Set}
...
 &&&& $J_{\text{\tiny JUMP}}(\boldsymbol{\mu}) \equiv \boldsymbol{\mu}_\mathbf{s}[0] $ \\
-&&&& This has the effect of writing said value to $\boldsymbol{\mu}_{pc}$. See section \ref{ch:model}. \\
+&&&& This has the effect of writing said value to $\boldsymbol{\mu}_{pc}$. See section \ref{ch:model}.\\
 \midrule
-0x57 & {\small JUMPI} & 2 & 0 & Conditionally alter the program counter. \\
+\linkdest{JUMPI}0x57 & {\small JUMPI} & 2 & 0 & Conditionally alter the program counter. \\
```
Comment (pirapira) [inline Paper.tex]: There should be a `{}` as a second argument.  Currently, the first `0` in `0x57` is considered the link target.

Comment (jamesray1) [reply Paper.tex]: If I separate them with a space, that should fix it. `{}` as a second argument isn't necessary, according to the macro.

**Pull request merged**